lourdesmorales-lvmor/week-3/giffaw-master/scripts/app.js

```diff
@@ -17,7 +17,7 @@ $("form").on("submit", function(e){
       console.log("success got data", response);
       var giphyData = response.data;
 
-      for(i in giphyData){
+      for(var i in giphyData){
```

Comment:
good use of "for in" very semantic. I also recommend using for each since that is more standard out in the wild. 
```
giphyData.forEach((gif)=>{`$(".gif-gallery").append(`<img src="${gif.images.original.url}"/>`)})
```